konfera/tests/test_models.py

```diff
@@ -133,6 +133,27 @@ def test_left_to_pay2(self):
         entry = models.Order(price=155.5, discount=5.5, amount_paid=200)
         self.assertEqual(entry.left_to_pay, 0)
 
+    def test_order_event_with_ticket(self):
+        entry = models.Order.objects.create(price=155.5, discount=5.5)
+        # Create Event
+        title = random_string(16)
+        slug = slugify(title)
+        date_to = timezone.now()
+        date_from = date_to + datetime.timedelta(days=1)
```

Comment:
Why is `date_from` after `date_to` ? 

Or a better question, but not directly to this PR . How is it possible it doesn't raise a `ValidationError` https://github.com/iraquitan/django-konfera/blob/59913d48efad30ee6a5f8de58b82ac67dbb66d9e/konfera/models/abstract.py#L33

@ricco386 I think `FromToModel`'s `.save` method needs to invoke `.clean()`, am I right?